docs/en/action-menu-items/custom-browser-protocols.md

```diff
@@ -162,41 +163,51 @@ By varying the keyword after the `//` part of the URL in your AMI, you can chang
 
 Using this methodology you could open applications, upload content via services like FTP, archive data, send email, or generate PDF reports.
 
-## Registering a protocol on OSX
+## Registering a protocol on macOS
+
+**Create and configure an `.app` bundle**
```

Comment:
Can you remove this subtitle? I don't think it's useful as the text below is a good introduction for "Registering a protocol on macOS"

Reply:
Ah, yes. Can do!

Reply:
It's removed, and I'll publish as soon as the preview looks good. Thanks so much for all your feedback, @julien-lang !